conans/tools.py

```diff
@@ -15,7 +15,7 @@
 from conans.client.output import ConanOutput
 # Tools from conans.client.tools
 from conans.client.tools import files as tools_files, net as tools_net, oss as tools_oss, \
-    system_pm as tools_system_pm, win as tools_win
+    system_pm as tools_system_pm, win as tools_win, settings as tools_settings
```

Comment:
```suggestion
    system_pm as tools_system_pm, win as tools_win
```

(see comment below)

Reply:
We don't need to import `settings` as `tools_settings`, all functions are already imported.